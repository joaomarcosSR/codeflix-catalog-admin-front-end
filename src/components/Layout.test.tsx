import { Typography } from "@mui/material";
import { render } from "@testing-library/react";
import { Layout } from "./Layout";

describe("Layout", () => {
  it("should render Layout correctly", () => {
    const { asFragment } = render(
      <Layout>
        <Typography>Ol√°</Typography>
      </Layout>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
