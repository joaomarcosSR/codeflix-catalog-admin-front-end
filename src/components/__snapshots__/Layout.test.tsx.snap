// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render Layout correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1km29jt-MuiContainer-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
      >
        Ol√°
      </p>
    </div>
  </div>
</DocumentFragment>
`;
